tests/garage/tf/exploration_strategies/test_epsilon_greedy_strategy.py

```diff
@@ -0,0 +1,37 @@
+"""
+This script creates a unittest that tests Gaussian policies in
+garage.tf.policies.
+"""
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.envs import TfEnv
+from garage.tf.exploration_strategies import EpsilonGreedyStrategy
+from garage.tf.policies import DiscreteQfDerivedPolicy
+from garage.tf.q_functions import DiscreteMLPQFunction
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyDiscreteEnv
+
+
+class TestEpsilonGreedyStrategy(TfGraphTestCase):
```

Comment:
try to write this as a unit test rather than an integration test. i don't think you need any TensorFlow dependency to test the epislon greedy strategy.